lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -0,0 +1,1234 @@
+        'fu_1_symptoms',
+        'fu_2_symptoms',
+        'fu_3_symptoms',
+        'fu_4_symptoms'
+    ]
+
+    for field in checkbox_fields:
+        record[field] = combine_checkbox_answers(record, field)
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference, False)
+
+def create_testing_determination_internal_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the
+    testing_determination_internal instrument. This instrument is used
+    to communicate with the Priority Queue application. Note that for this
+    questionnaire we do want to return answers for all questions even if they are
+    not answered.
+    """
+
+    string_questions = [
+        'testing_trigger',
+        'testing_type',
+        'surge_selected'
+    ]
+
+    question_categories = {
+        'valueString': string_questions
+    }
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference, False)
+
+def create_computed_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict,
+                                            birthdate: datetime, encounter_date: datetime) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for a
+    computed questionnaire. This "computed questionnaire" produces
+    answers that do not appear directly in an actual REDCap survey.
+    For example, a computed question captures the participant's age
+    on the date of the encounter.
+    """
+
+    record['age'] = relativedelta(encounter_date, birthdate).years
+
+    integer_questions = [
+        'age'
+    ]
+
+    question_categories = {
+        'valueInteger': integer_questions
+    }
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference, False)
+
+def create_daily_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the daily
+    attestation questionnaire.
+    """
+    boolean_questions = [
+        'daily_symptoms',
+        'prev_pos',
+        'hall_health_2',
+        'prior_test',
+        'travel_countries_phs',
+        'travel_states_phs',
+        'screen_positive'
+    ]
+
+    string_questions = [
+        'daily_exposure',
+        'daily_exposure_known_pos',
+        'sought_care',
+        'hospital_where',
+        'hospital_ed',
+        'prior_test_positive',
+        'prior_test_type',
+        'prior_test_result',
+        'contact',
+        'wash_hands',
+        'clean_surfaces',
+        'hide_cough',
+        'mask',
+        'distance',
+        'attend_event',
```

Comment:
This should be a boolean, like the question on pregnancy

Reply:
attend_event choices are:
yes, Yes
no, No
dont_say, Prefer not to say
That's why I made it a string.

Reply:
These are the same values we see for `core_pregnant`, but we're ingesting `core_pregnant` as boolean. Therefore we should be consistent.

Reply:
> Is having the symptoms enough?

Probably.

Reply:
That is a good argument! Changed in latest commit.